rules/aws_cloudtrail_rules/aws_rds_snapshot_shared.py

```diff
@@ -0,0 +1,36 @@
+from panther_base_helpers import aws_rule_context
+
+# Account IDs exempted from this rule
+ALLOWED_ACCOUNTS = {}
+
+
+def rule(event):
+    if all(
+        [
+            event.get("eventSource", "") == "rds.amazonaws.com",
+            event.get("eventName", "") == "ModifyDBSnapshotAttribute"
+            or event.get("eventName", "") == "ModifyDBClusterSnapshotAttribute",
+            event.deep_get("requestParameters", "attributeName") == "restore",
+        ]
+    ):
+        shared_account_ids = event.deep_get("requestParameters", "valuesToAdd", default=[])
```

Comment:
Can this be compared to userIdentity.accountId and only alert if the valuesToAdd account ID is different?

Reply:
I don't know if a snapshot can be shared with the account it's created in. We can certainly add the logic to check for it, though.

Reply:
Updated in https://github.com/panther-labs/panther-analysis/pull/1036/commits/58b0ad869a3eae3609dfd80f0073e30df35ed0c0.